<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>412</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_left">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_table">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tree">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="button_maketree">
          <property name="text">
           <string>Построить дерево объектов анализа</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tree2">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_right">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_data">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_data">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_filter"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Открыть</string>
     </property>
     <addaction name="action_add_csv"/>
     <addaction name="action_add_xml"/>
     <addaction name="action_add_xls"/>
     <addaction name="action_DB"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_settings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add_csv"/>
   <addaction name="action_add_xml"/>
   <addaction name="action_add_xls"/>
   <addaction name="action_DB"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_add_csv">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/csv.png</normaloff>:/all/Resources/csv.png</iconset>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="action_add_xml">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/xml.png</normaloff>:/all/Resources/xml.png</iconset>
   </property>
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="action_add_xls">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/xls.png</normaloff>:/all/Resources/xls.png</iconset>
   </property>
   <property name="text">
    <string>XLS</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/power-button.png</normaloff>:/all/Resources/power-button.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/switch.png</normaloff>:/all/Resources/switch.png</iconset>
   </property>
   <property name="text">
    <string>Настройки файла</string>
   </property>
  </action>
  <action name="action_DB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/all/Resources/db.png</normaloff>:/all/Resources/db.png</iconset>
   </property>
   <property name="text">
    <string>База данных</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
